Sources/MapboxCoreNavigation/CongestionLevel.swift

```diff
@@ -97,4 +105,34 @@ extension RouteLeg {
 
         return congestionLevels
     }
+    
+    /**
+     Return an array containing the traffic congestion level along each road segment in the route leg geometry.
+
+     The array is formed either by converting values of `segmentNumericCongestionLevels` to `CongestionLevel` type by congestionMapping,
+     or by taking `segmentCongestionLevels`, depening whether `AttributeOptions.numericCongestionLevel` or `AttributeOptions.congestionLevel`
+     was specified in `DirectionsOptions.attributes` during Directions request.
+
+     - parameter congestionMapping: The `CongestionMapping` closure to map `NumericCongestionLevel` and `MapboxStreetsRoadClass` to `CongestionLevel`.
+     - returns: An array containing the traffic congestion level along each road segment in the route leg geometry.
+     */
+    public func resolvedCongestionLevels(congestionMapping: CongestionMapping? = nil) -> [CongestionLevel]? {
```

Comment:
Before publicly exposing this implementation would it make sense to align with Android SDK regarding this? Do they have something similar at this point? 

/cc @mapbox/navigation-android

Reply:
I think not. The corresponding method is for internal use only.
However, we have an API that allows setting custom numeric ranges for any of the `CongestionLevel`: https://github.com/mapbox/mapbox-navigation-android/blob/main/libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/route/line/model/RouteLineColorResources.kt#L300